### ISSUE TITLE
Cannot modify/delete an entity or add a newer

### ISSUE THREAD

**Issue by Emic37** [opengisch/QField#692]
I encounter a really big bug using both beta-apks or Matterhorn version : I can't add anymore a new entity or modify an existing entity. 

After modification or adding, all happens as if there is no problem. But after saving changes, all entities disappears of the map. The only way I found to retrieve the entities is to relaunch QField. The entities appears again but all changes are lost. 

### Reproduction Steps:

1. Modify or add an entity,
2. Fill a few attributes,
3. Save the change.

### Expected behavior:

Simply save changes or adding.

### Observed behavior:

All happens as if there is no problem but when we come back on the map, all entities disappeared. And when we relaunch QField, existing entities appears again but the changes which made on the attributes are lost. Likewise if we try to add a new entity, the new entity is not recorded.

Qfield returns this error : `Feature %1 could not be fetched after commit`. I have this issue using my project of course. But I tested with the "bees_offline" demo project and I encounter a same behavior except to update entities.

### Screenshots and GIFs

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/2921424/67600630-c2eaa200-f772-11e9-8874-207466606acc.gif)

![ezgif com-optimize](https://user-images.githubusercontent.com/2921424/67601445-af403b00-f774-11e9-9a07-cec8f324dc3a.gif)

The entities are displayed again after reboot or reload project, but changes are lost. The problem seems disappear when the 'show feature count' option is disabled in the QGIS project. That seems to be similar than #688.

**QField version:** matterhorn 1.2.0 release and beta-apk 10200995

## Additional information:

* Problem started happening recently, didn't happen in an older version of QField: [tested with 1.1.1 and 1.1.0, same problem]
* Problem can be reliably reproduced, doesn't happen randomly: [Yes]
* Problem happens with all files and projects, not only some files or projects: [Yes]

**Comment by rwjdennis** [comment]
I think this is the same as the issue I posted (https://github.com/opengisch/QField/issues/688). @Emic37 , it looks like you have checked the "Show Feature Count" box in the desktop, as in the second .gif you posted the feature counts are displayed in the layers panel. Uncheck the box on the desktop project, re-export it to your device and then try adding points in QField.

**Comment by Emic37** [comment]
@rwjdennis Oh ! Thank you very much. I just tested to disable the "show feature count" option and it works again. I started to be worried because I'll go to the field in the future days and the title of your issue didn't attract my attemption.

I didn't focus on your issue because I use QField with this option enabled without problem since few months . Or I dream... That's right, I've made some changes in my project these last weeks. Strange... 
In brief, thanks !
